.github/workflows/ci.yml

```
@@ -218,5 +221,18 @@ jobs:
         uses: actions/download-artifact@v2
         with:
           name: nuget-packages
+
+      # if tag cointains suffix "-" treat it as pre-release
```

MINOR: Spelling of `cointains`. Also `-` is not a suffix.

```suggestion
      # if tag contains "-" treat it as pre-release
```